VEHICLE/SetConvertibleRoof.md

```diff
@@ -7,9 +7,39 @@ ns: VEHICLE
 // 0xF39C4F538B5124C2 0xC87B6A51
 void SET_CONVERTIBLE_ROOF(Vehicle vehicle, BOOL p1);
```

Comment:
Maybe rename `p1` to `toggle`, or `roofState`, but I believe this toggles a roof on/off.